### PR TITLE
[Doc Improvement][Remove deprecated "packageName" value from app manifest schema article][2651838]

### PR DESCRIPTION
**Pull request by v-ajiths**
Edited and removed the occurrence of packageName in the article.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8da0156d032c8dcc542960f653228b7fc31644b9/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 61 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ab07c70b-c826-4cd0-9fd3-6bfb0b469b35) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/universal-actions-for-search-based-message-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8da0156d032c8dcc542960f653228b7fc31644b9/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Funiversal-actions-for-search-based-message-extensions.md) | 96 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/39988043-bc13-4f97-9232-6e0650cb5ff7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8da0156d032c8dcc542960f653228b7fc31644b9/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 89 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/11c1058c-b966-4429-95f5-dfcab60352bf) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/search-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8da0156d032c8dcc542960f653228b7fc31644b9/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fsearch-extensions.md) | 90 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5b410c11-253a-45d7-9406-72af9c0d83bd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8da0156d032c8dcc542960f653228b7fc31644b9/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/888efaa6-4602-48cb-93bc-7cfe5b2ac0c2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8da0156d032c8dcc542960f653228b7fc31644b9/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c939dff3-e4b4-4206-be00-a555e993b963) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8da0156d032c8dcc542960f653228b7fc31644b9/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-manifest.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ade4ce50-105a-48f6-b71a-84302eb308fa) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/build-adaptive-card-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8da0156d032c8dcc542960f653228b7fc31644b9/msteams-platform%2Ftabs%2Fhow-to%2Fbuild-adaptive-card-tabs.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/79e1d3f1-d9d0-45b4-8269-a6d3aa55d3af) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8da0156d032c8dcc542960f653228b7fc31644b9/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f862bae8-cd6b-4fd6-94ae-2d73c1dded8b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a88c507275f1409b1b01793fefd45589b63cd02d/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 61 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da11543e-52a4-4446-8e7d-739a6acd6cd1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/universal-actions-for-search-based-message-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a88c507275f1409b1b01793fefd45589b63cd02d/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Funiversal-actions-for-search-based-message-extensions.md) | 96 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c699f8ec-80a4-440f-8a99-faf432743b20) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a88c507275f1409b1b01793fefd45589b63cd02d/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/180267fe-4f1b-4d93-91de-87c21627b683) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/search-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a88c507275f1409b1b01793fefd45589b63cd02d/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fsearch-extensions.md) | 90 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1bb309ce-619f-441e-a0b1-637d721a1f04) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a88c507275f1409b1b01793fefd45589b63cd02d/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cd2b5350-dff2-4bfc-a901-3934a6685ebd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a88c507275f1409b1b01793fefd45589b63cd02d/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/024910e4-ea79-4618-9db7-84c3efd33bb6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a88c507275f1409b1b01793fefd45589b63cd02d/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-manifest.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/19eb3a9c-c006-4de6-9963-b41ebdff9a1d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/build-adaptive-card-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a88c507275f1409b1b01793fefd45589b63cd02d/msteams-platform%2Ftabs%2Fhow-to%2Fbuild-adaptive-card-tabs.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/996517bf-a02f-4456-8834-87f509e6a71f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a88c507275f1409b1b01793fefd45589b63cd02d/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ce19c530-ea21-478a-9f4e-50391c1abb6f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b150902afef24530169035459fce0c81977abfd4/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 61 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/219363d5-988a-4ef5-95d2-daf5832cd316) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/universal-actions-for-search-based-message-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b150902afef24530169035459fce0c81977abfd4/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Funiversal-actions-for-search-based-message-extensions.md) | 96 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/99017f93-7d81-434c-9eee-bdd4ba95e682) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b150902afef24530169035459fce0c81977abfd4/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fb494ace-e264-4fa4-b7ae-f4f214bcba4c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/search-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b150902afef24530169035459fce0c81977abfd4/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fsearch-extensions.md) | 90 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8acb6ed8-3dd8-414f-ba0f-085f1e49e352) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b150902afef24530169035459fce0c81977abfd4/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b96e4d23-713d-4ad3-b5a6-6d6d7ddbaafe) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b150902afef24530169035459fce0c81977abfd4/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a4ff3620-8f0f-411a-a0fc-3bc97c51e3bb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b150902afef24530169035459fce0c81977abfd4/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-manifest.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1aed90e3-2cb6-4547-8573-6c4082bfe10d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/build-adaptive-card-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b150902afef24530169035459fce0c81977abfd4/msteams-platform%2Ftabs%2Fhow-to%2Fbuild-adaptive-card-tabs.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ec6bc696-16c4-4c39-97bd-92833748dae5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b150902afef24530169035459fce0c81977abfd4/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4f83a4fa-b595-40b9-a44b-95009857bbb9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c8efcee6c043d0e55a9365d53ce09f7a7f41c61/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 61 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/769fdb67-13e1-4cf1-b82b-888cff4f6078) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/universal-actions-for-search-based-message-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c8efcee6c043d0e55a9365d53ce09f7a7f41c61/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Funiversal-actions-for-search-based-message-extensions.md) | 96 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fb2002dd-efc3-4c5c-88b0-659e1ee2eb26) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c8efcee6c043d0e55a9365d53ce09f7a7f41c61/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f85f2d53-41c5-455b-9788-b56b20d894dc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/search-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c8efcee6c043d0e55a9365d53ce09f7a7f41c61/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fsearch-extensions.md) | 90 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2ceeb008-7cc4-4f48-8145-4dbaebaa7bca) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c8efcee6c043d0e55a9365d53ce09f7a7f41c61/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/27e9d432-1fca-42a9-be77-7936f86dba4d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c8efcee6c043d0e55a9365d53ce09f7a7f41c61/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/696fdf12-9c64-48c3-9ae6-1b4cb5485c9f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c8efcee6c043d0e55a9365d53ce09f7a7f41c61/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-manifest.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aaa65d46-d7c7-4c74-8320-608e1ce42a2c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/build-adaptive-card-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c8efcee6c043d0e55a9365d53ce09f7a7f41c61/msteams-platform%2Ftabs%2Fhow-to%2Fbuild-adaptive-card-tabs.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8773cf79-a9ef-4329-b0b9-8f9d66b0a797) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c8efcee6c043d0e55a9365d53ce09f7a7f41c61/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2449cabd-6da2-4671-939e-d997b426099e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fb70cde616b1a1f28ba70923fa3b4be9898d56/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 61 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9663ed44-70b9-46a5-88f6-f7cfb58976a4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/universal-actions-for-search-based-message-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fb70cde616b1a1f28ba70923fa3b4be9898d56/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Funiversal-actions-for-search-based-message-extensions.md) | 96 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/87bf6acc-6468-4cb3-ab5d-8d9c8ab114c6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fb70cde616b1a1f28ba70923fa3b4be9898d56/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ce1e4549-dfc7-4ff0-8f16-e1c119378718) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/search-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fb70cde616b1a1f28ba70923fa3b4be9898d56/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fsearch-extensions.md) | 90 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ec5c040-8639-439c-a757-33cae758a0de) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fb70cde616b1a1f28ba70923fa3b4be9898d56/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/64da3b7c-4b59-400d-9369-568a5bfd37f4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fb70cde616b1a1f28ba70923fa3b4be9898d56/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/24a08b9e-7261-47f3-b18d-8e667bdd2fef) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fb70cde616b1a1f28ba70923fa3b4be9898d56/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-manifest.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a09e33a0-09f2-45d1-b5c0-9caef00711c0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/build-adaptive-card-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fb70cde616b1a1f28ba70923fa3b4be9898d56/msteams-platform%2Ftabs%2Fhow-to%2Fbuild-adaptive-card-tabs.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f2d507ae-0e11-4171-a4e3-72ecfc1b9ce6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fb70cde616b1a1f28ba70923fa3b4be9898d56/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/90edb655-47de-437e-ac50-072785e0a074) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/messaging-extension-v3/create-extensions.md [hunk 1]
```diff
@@ -127,7 +124,7 @@ To initiate actions from a message extension, set the `type` parameter to `actio
 
 ### Initiate actions from messages
 
-In addition to initiating actions from the compose message area, you can also use your message extension to initiate an action from a message. This will allow you to send the message contents to your bot for processing and optionally reply to that message with a response using the method, which is described in [Responding to submit](#responding-to-submit). The response will be inserted as a reply to the message that your users can edit before submitting. Your users can access message extension from the overflow `...` menu and then selecting `Take action` as in the following image:
+You can initiate actions from the compose message area and also use your message extension to initiate an action from a message, which allows you to send the message contents to your bot for processing. Optionally, you can reply to that message with a response using the method described in [Responding to submit](#responding-to-submit). The response will be inserted as a reply to the message that your users can edit before submitting. Your users can access message extension from the overflow `...` menu and then selecting `Take action` as in the following image:
```
Comment (AkJo) [inline msteams-platform/resources/messaging-extension-v3/create-extensions.md:127]: This particular line does not seem to make much sense. We should rephrase it.

**Pull request merged**